CMakeLists.txt

```diff
@@ -1,5 +1,83 @@
 esma_set_this ()
 
+option(BUILD_GEOS_GTFV3_INTERFACE "Build GEOS-gtFV3 interface" OFF)
```

Comment:
@pchakraborty Can most of this new chunk be moved lower in the CMakefile.    I'm concerned that the typical user comes here just to add/delete a source file and now has to scan down.   If at least those sources could be moved up.

Reply:
Yeah. Might be best if the main `srcs` list was still at the top. That's probably what most people will be looking for in here...

Reply:
Looks good to me. I'll let @tclune weigh in. :)